packages/jsii-pacmak/test/expected.jsii-calc/dotnet/Amazon.JSII.Tests.CalculatorPackageId/Amazon/JSII/Tests/CalculatorNamespace/ClassThatImplementsTheInternalInterface.cs

```diff
@@ -2,7 +2,9 @@
 
 namespace Amazon.JSII.Tests.CalculatorNamespace
 {
-    /// <remarks>stability: Experimental</remarks>
+    /// <remarks>
+    /// stability: Experimental
+    /// </remarks>
     [JsiiClass(nativeType: typeof(ClassThatImplementsTheInternalInterface), fullyQualifiedName: "jsii-calc.ClassThatImplementsTheInternalInterface")]
     public class ClassThatImplementsTheInternalInterface : DeputyBase, IINonInternalInterface
```

Comment:
I see we are implicitly implementing the interfaces. What happens when we implement 2 interfaces with the same method signatures? Can't we always explicitly implement?

Reply:
You mean on the inherited/implemented classe/interfaces? 

Sure, I can make a change to just always put the fully qualified type.

Reply:
Just for reference: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/interfaces/explicit-interface-implementation

Reply:
Actually, it looks like the work may be more involved, I don't want to delay this PR (Which the goal is to be a 1:1 against the OLD generator) so I created this issue to address the problem:

https://github.com/awslabs/jsii/issues/560